packages/backend/src/ee/services/ai/utils/getSlackBlocks.ts

```diff
@@ -213,6 +213,22 @@ export function getDeepLinkBlocks(
         return [];
     }
 
+    // Check if any artifacts have dashboard configs
+    const hasDashboard = artifacts.some((artifact) => artifact.dashboardConfig);
+
+    // Add prominent dashboard link if dashboard artifact exists
+    if (hasDashboard) {
+        return [
+            {
+                type: 'section',
+                text: {
+                    type: 'mrkdwn',
+                    text: `📊 <${siteUrl}/projects/${slackPrompt.projectUuid}/ai-agents/${agentUuid}/threads/${slackPrompt.threadUuid}|View Dashboard in Lightdash ⚡️>`,
+                },
+            },
+        ];
+    }
```

Comment:
We send a link to the thread, because agent doesn't generate a dashboard, it generates an artifact.

Would be nice to have a dedicated page for artifact? that an unfurl